.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,107 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
```

Comment:
@luca-rath in sulu we are changing to `2.x` for better composer integration so I would add that also to all repositories also the ones which do not yet have a `.x` branch yet. See also: https://github.com/sulu/sulu-docs/blob/3962e881c89d5dc6bfe76e4c8bd32985c0dfb6fa/.github/workflows/build-docs.yaml#L7

So I would go here currently with:

```
        branches:
            - master # only for them which have a master branch yet
            - develop # only for them which have a develop branch yet
            - '[0-9]+.x'
            - '[0-9]+.[0-9]+'
```